File: src/main/java/com/atsuishio/superbwarfare/client/model/entity/Sc250Model.java
package com.atsuishio.superbwarfare.client.model.entity;

import com.atsuishio.superbwarfare.Mod;
import com.atsuishio.superbwarfare.entity.projectile.Sc250Entity;
import net.minecraft.resources.ResourceLocation;
import software.bernie.geckolib.model.GeoModel;

public class Sc250Model extends GeoModel<Sc250Entity> {

    @Override
    public ResourceLocation getAnimationResource(Sc250Entity entity) {
        return null;
    }

    @Override
    public ResourceLocation getModelResource(Sc250Entity entity) {
        return Mod.loc("geo/sc_250.geo.json");
    }

    @Override
    public ResourceLocation getTextureResource(Sc250Entity entity) {
        return Mod.loc("textures/entity/ju_87.png");
    }
}

File: src/main/kotlin/com/atsuishio/superbwarfare/client/model/entity/Sc250Model.kt
package com.atsuishio.superbwarfare.client.model.entity

import com.atsuishio.superbwarfare.Mod.Companion.loc
import com.atsuishio.superbwarfare.entity.projectile.Sc250Entity
import software.bernie.geckolib.model.GeoModel

class Sc250Model : GeoModel<Sc250Entity>() {
    override fun getAnimationResource(entity: Sc250Entity) = null

    override fun getModelResource(entity: Sc250Entity) = loc("geo/sc_250.geo.json")

    override fun getTextureResource(entity: Sc250Entity) = loc("textures/entity/ju_87.png")
}
